### ISSUE TITLE
Push new notes into view

### ISSUE THREAD

**Issue by mhkeller** [documentcloud/documentcloud#246]
I'm currently doing some team document annotation with @joannaskao and we find ourselves refreshing the page to see if the other one as already annotated an interesting part or not. This wouldn't be too bad but the scroll also seems to take us back to the top of the doc so we have to go find where we were again.

It's a bit of a heavy lift but putting it on the radar that some type of live push for new notes would be awesome. Or in the short term, some kind of scroll position persistence (but I'm conflicted on that one since that doesn't strike me as an across-the-board expected behavior and all-around improvement).

**Comment by reefdog** [comment]
Agree, this sort of live update is sort of expected behavior these days. Unsure when we'll tackle it, but it makes sense.

In the meantime, to lower the refresh cost a little, you could click the little page indicator at the top left of the page you're on before refreshing. That'll anchor the viewport to that page (and change the URL to reference it, like [so](https://www.documentcloud.org/documents/282753-lefler-thesis.html#document/p9)) so that upon refresh you'll at least be right back on that page. Not the most elegant of workarounds, but might save you a bit of scrolling.

**Comment by mhkeller** [comment]
Ah good call. Thanks!
